2-mandatory/3-bush-berries.js

```diff
@@ -22,9 +22,15 @@
 */
 
 function isBushSafe(berryArray) {
-  //Write your code here
+  if (berryArray.every(berry => berry === "pink")) {
+    return "Bush is safe to eat from";
+  }
+else {
```

Comment:
Do you need the additional else here?
Maybe something like below
`  
if (berryArray.every(berry => berry === "pink")) {
    return "Bush is safe to eat from";
  }

return "Toxic! Leave bush alone!";

`